lib/routemaster/api_client.rb

```diff
@@ -8,6 +8,7 @@
 require 'routemaster/middleware/error_handling'
 require 'routemaster/middleware/metrics'
 require 'routemaster/responses/response_promise'
+require 'routemaster/resources/rest_resource'
```

Comment:
Can we keep a comment about bad dependencies here?
Otherwise we'll have the issue again next time someone cleans up the require tree.